### PR TITLE
added vidadeprogramador

### PR DESCRIPTION
**Pull request by caio2k**
Hello there,
Thanks for your great job creating this app. I've read the guideline to add a comic and I'm adding 2 local brazilian comics into your app but as I don't have yet the environment ready to compile I cannot test the build. Please evaluate if I need it tested before creating a PR.
Regards.

### CODE REVIEW

File: plugins/vidadeprogramador/extract.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function(page) {
+    var regex = /<img[^>]*src="([^"]*\/uploads\/[^"]*\/tirinha[^"]*.png)"/;
```
Comment (tardypad) [inline plugins/vidadeprogramador/extract.js]: Unless there is a good reason for it, do not match the ".png" extension.
That way if the page change to use ".jpg" for example, the comic extraction wouldn't break.

File: plugins/vidadesuporte/extract.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ unction(page) {
+    var regex = /<img[^>]*src="([^"]*\/wp-content\/uploads\/[^"]*\/Suporte_[^"]*.jpg)"/;
```
Comment (tardypad) [inline plugins/vidadesuporte/extract.js]: Same as before, do not match the ".jpg" extension

**Comment by tardypad** [comment]
Thanks for the PR

A few comments besides the ones in the review:
- please also provide a 1000x1000 pixels cover.big.jpg just like the other ones have
- if you can, split the commit to have each comic added in its own.
Ideally they would be in 2 separate PR too to make it easier to review and merge.
- in the cover of "videdesuporte", display a single character without any object around it.

**Comment by caio2k** [comment]
hey @tardypad i've updated the PR as you suggested. please take a look. whenever you accept it I will push the other PR with the next comics.

**Review by tardypad — CHANGES_REQUESTED** [review]

**Comment by tardypad** [comment]
Thanks for the changes, it looks fine now.
However there is one more small item to fix because I forgot to press submit to the review previously...
You can already open the other PR so I can look at them.

File: plugins/vidadeprogramador/extract.js (hunk 1 above)
Comment (caio2k) [reply plugins/vidadeprogramador/extract.js]: ok, done

**Comment by Olf0** [comment]
This is ready to be merged for 9 months, isn't it?

**Comment by caio2k** [comment]
Yes, it is there for a while. I could not open a new PR because there is one file that cannot be forked properly...

**Review by Maledictus — APPROVED** [review]

**Comment by Maledictus** [comment]
Thank you for your efforts

**Pull request merged**